style.css

```diff
@@ -0,0 +1,186 @@
+body{
+    margin: 3%;
+    background-image: url("assets/background.png");
+    background-repeat: no-repeat;
+    background-size:cover;    
+}
+.website-title{
+    color:white;
+    font-size: 57px;
+    font-family:Monotype Corsiva, Times, serif;
+}
+ul {
+ list-style-type: none; 
+    margin: 0;
+    padding: 0;
+ }
+
+ nav .links li{
+    display: inline-block;
+}
+ul.links a{
+    color:white;
+    text-decoration: none;
+    display:inline-block;
+    text-align: center;
+    background:-webkit-linear-gradient(white 5%, gray 10%, #141414 80%, black 10%);
+    background-color:black;
+    color: white;
+    width: 150px;
+    height: 30px;
+    margin: 1px;
+    border-bottom: none;
+    border-right: none;
+    border-top: none;
+    border-left-width: 1px;
+    font-size: 20px;
+    margin-bottom: 20px;
+}
+a {
+    text-decoration: none !important;
```

Comment:
in general, don't use !important - if you work in a team and use !important, people will hate you. Just use a class name to your a tags, and use that class name for styling